frontend/src/components/Toplist.tsx

```diff
@@ -0,0 +1,100 @@
+const Toplist = () => {
+    return (
+        <>
```

Comment:
Il componente deve essere dinamico, prendere dati dalle API e iterarli per costruirci la table al di sopra. Non devono esserci dati statici come avviene in questo caso. Se  hai difficoltà chiedi a Marco come richiamare le api